backend/booking_engine/README.md

```diff
@@ -1,49 +1,139 @@
-![Chronogolf][crest]
+![Lightspeed Golf][crest]
 
 # The Booking Engine challenge
 
-Golf clubs manage player reservations through our system. When a player wants to
+## Intro to Building a Booking Engline
+You can use this prompt to introduce the the challenge and explain a little about our app:
+> Golf clubs manage player reservations through our system. When a player wants to
 make a reservation, they request it through a booking engine. The system will
 then confirm if the submitted reservation is valid.
-
-Your mission today consists of building a simple booking application that
+>
+> Your mission today consists of building a simple booking application that
 validates reservation requests and returns a list of confirmed reservations.
 
-### Environment Setup
+</br>
 
-Let's take care of your setup first.
+As you interview, be mindful of the following things overall:
 
-- Choose a dev environment and a language that suits you
-  - A dev environment on your personal computer is perfect!
-  - Coderpad is available if you would prefer
-  - Choose the language you are most comfortable with
-- Let's start pairing together!
+- Is the candidate a person we would enjoy working with?
+- Is the candidate receptive to feedback?
+- Did the candidate showcase any toxic or inappropriate behaviour?
+- Will the candidate be interested in the work and culture we can provide?
 
-### Step 1 - Be able to create a new reservation
+#
 
-Before anything else, we want to be able to accept new bookings from players
-Provide a method to create a new reservation
+## Environment Setup
 
-### Step 2 - Validate reservations are 10 minutes apart
+You can use this prompt to align on what environment will be used for the challenge:
+> Let's take care of your setup first.
+> - Choose a dev environment and a language that suits you
+>   - A dev environment on your personal computer is perfect!
+>   - Coderpad is available if you would prefer
```

Comment:
AFAIK we actually don't have a Coderpad license, but instead one for [codeinterview.io](http://codeinterview.io/) org wide. We should properly update this to specify.